src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -169,8 +183,11 @@ export class BucketDetailComponent implements OnInit {
                   folderContain = true;
                 }else if(ObjectKeyNum == folderNum + 1){
                   //Identify folders within folders
+                  alldir 
                   let lastNum = arr.Contents.Key.lastIndexOf(this.colon);
                   if(lastNum == arr.Contents.Key.length -1){
+
```

Comment:
remove white space.

Reply:
I have remove it, plz check again,thank you.